wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WireExtension.kt

```diff
@@ -351,10 +352,16 @@ open class WireExtension(
     }
 
     /** Sets a project. */
+    @Deprecated("Use srcProject(ProjectDependency) instead. This method will be removed in a future version of Wire.")
```

Comment:
Make it DeprecationLevel.HIDDEN ?

Reply:
done!

Reply:
I forgot to push...

https://github.com/square/wire/pull/3124